### DISCUSSION TITLE
How to wrap the column header text?

### DISCUSSION THREAD

**Discussion by zaqxs123456** [revolist/revogrid#656] [General]
How to change the height of the header and wrap the header text like this?

![Screenshot_20211209_140246](https://user-images.githubusercontent.com/56287342/145343624-6a80b2b6-1169-4e4e-92cc-cb92b5895d24.png)

**Comment by van-huyen** [comment]
Hi @zaqxs123456 

this is css control, not of revo-grid.

First, set row header height up to 2-3 line (50px):
 revo-grid[theme='default'] revogr-header .header-rgRow {
    **height: 50px;**
}

Second, add wrap text style:
   **white-space: pre-wrap;**

![image](https://user-images.githubusercontent.com/83264685/148471486-ee8ef4d8-65fa-4607-9bf6-f091849394d9.png)
